src/components/Contact.tsx

```diff
@@ -7,34 +7,34 @@ import { useTranslation } from "react-i18next";
 
 type ContactPageProps = SpaceProps & TypographyProps;
 
-const Container = styled.div<ContactPageProps>`
-  ${typography};
-  text-transform: uppercase;
-`;
-
 const H1 = styled.h1<ContactPageProps>`
   ${space};
   ${typography};
+  text-transform: uppercase;
 `;
 
 const ContactInfo = styled.p<ContactPageProps>`
   ${space};
   ${typography};
+  text-transform: uppercase;
 `;
 
 const Contact = () => {
   const { t } = useTranslation();
-  const fontSizes = [2, 3, 4, 5];
+  const contactInfo = t("contact.contact-info");
+
   return (
     <Flex flex="auto" alignItems="center">
-      <Container>
+      <div>
         <H1 fontSize={[4, 5, 6, 7]} py={3}>
           {t("contact.header")}
         </H1>
-        <ContactInfo fontSize={fontSizes} py={3}>
-          {t("contact.contact-info")}
-        </ContactInfo>
-      </Container>
+        <ContactInfo
+          fontSize={[4, 4, 4, 5]}
+          py={3}
+          dangerouslySetInnerHTML={{ __html: contactInfo }}
```

Comment:
so id be wary of using `dangerouslySetInnerHTML` where its not necessary, you could achieve the desired effect by having 2 `ContactInfo` tags (which are p tags) rather than 1

Reply:
good to know as a future ref, i've just updated that now

Reply:
oops forgot to add the translation, one sec

Reply:
wait there is no translation, i'm going mad! ignore me